### ISSUE TITLE
All varImp is 0

### ISSUE THREAD

**Issue by ChungsooKim** [OHDSI/DeepPatientLevelPrediction#18]
As @jreps mentioned prior meeting (21-10-04), 

I also found that all variable importance of the fitted model is zero. 

![image](https://user-images.githubusercontent.com/42205861/137727731-d6680a2e-c2fe-4b6c-857a-21ce89073f1b.png)

However, I also figured out the final model from the fitDeepNNTorch function still had values of predictors as weights and biases in each layer. so we need to discuss how to deal with these values for model reproducibility

```R
model$parameters()
```
![image](https://user-images.githubusercontent.com/42205861/137728273-0edde857-7795-4ebe-8a2f-123e21787f95.png)

![image](https://user-images.githubusercontent.com/42205861/137728330-97ea550c-cbc0-4932-a8d7-8e8a797f7ff8.png)

**Comment by ted9219** [comment]
Yes, that's right. we will use millions of covariates from millions of patients.
Then we can store all the original values of the weights for each layer in a separate column. 
However, if we prepare the code for calculating the importance metric in advance, users can only calculate the metric if they want (optionally). I would try to get agnostic codes for the torch.
